codecs/avif_dec/build.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -e
+
+export OPTIMIZE="-Os"
+export LDFLAGS="${OPTIMIZE}"
+export CFLAGS="${OPTIMIZE}"
+export CPPFLAGS="${OPTIMIZE}"
+
+echo "============================================="
+echo "Compiling libaom"
+echo "============================================="
+test -n "$SKIP_LIBAOM" || (
+  cd node_modules/libavif/ext  
+  test -d aom || git clone -b v1.0.0-errata1-avif --depth 1 https://aomedia.googlesource.com/aom aom
```

Comment:
I've noticed we use `napa` in other codecs, but manual `git clone` here. I don't have preference either way, but probably best to stick to one method for all? Which one would it be?

Reply:
I prefer napa. I _think_ I went with a manual clone here for `depth`. Not sure. Feel free to switch.